docs/devGuide/development/settingUp.md

```diff
@@ -16,7 +16,7 @@ This page explains how to set up your development environment to start contribut
 ## Prerequisites
 
 1. **Node.js** ({{ node_dev_version }} or higher) with<br>
-   **npm** <tooltip content="You may need to upgrade the default npm that is included with Node.js v14 installation">v7</tooltip> or higher
+   **npm** <tooltip content="You may need to upgrade the default npm that is included with Node.js v16 installation">v8</tooltip> or higher
```

Comment:
Is the tooltip still relevant?

Reply:
Mm I dont think so! Deleted in [46a9407](https://github.com/MarkBind/markbind/pull/2233/commits/46a94072c55083fc556a1af96a7c3570e000b99f)